activerecord/lib/active_record/relation.rb

```diff
@@ -507,6 +507,31 @@ def delete_all
       affected
     end
 
+    # Finds and destroys all records matching the specified conditions.
+    # Returns the collection of objects that were destroyed; each will be frozen, to
+    # reflect that no changes should be made (since they can't be persisted).
+
+    # If no record is found, returns empty array.
+    #
+    #   Person.all.destroy_by(name: 'Spartacus', rating: 4)
+    #   Person.all.destroy_by("published_at < ?", 2.weeks.ago)
+    #   Person.where(["user_name = ?", user_name]).destroy_by(name: 'Spartacus')
+    def destroy_by(arg, *args)
+      where(arg, *args).destroy_all
```

Comment:
Why not just `*args`? What's the purpose of splitting the first one out?

Reply:
Just `*args` make sense. Fixed it. 👍